pkg/vault/vault_test.go

```diff
@@ -0,0 +1,219 @@
+// Copyright (c) WarnerMedia Direct, LLC. All rights reserved. Licensed under the MIT license.
+// See the LICENSE file for license information.
+package vault
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	matchers "github.com/Storytel/gomock-matchers"
+	"github.com/golang/mock/gomock"
+	"github.com/hashicorp/vault/api"
+	"github.com/hbocodelabs/infratest/mock"
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	const_expected_key   = "key"
```

Comment:
Per my previous comment about `snake_case`, I'd expect these to be `expectedKey`, etc.

Reply:
Fixed!